apps/scan/frontend/src/screens/election_manager_screen.tsx

```diff
@@ -42,7 +42,7 @@ import {
 } from '../api';
 import { usePreviewContext } from '../preview_dashboard';
 import { LiveCheckButton } from '../components/live_check_button';
-import { CastVoteRecordSyncReminderModal } from '../components/cast_vote_record_sync_modal';
+import { CastVoteRecordSyncRequiredModal } from './cast_vote_record_sync_required_screen';
```

Comment:
As noted in the previous commit, this is:

```
/**
 * A secondary modal for explaining that an action is blocked until cast vote records have been
 * synced. Nudges users toward the primary {@link CastVoteRecordSyncRequiredScreen}.
 */
```

Not the main `CastVoteRecordSyncRequiredScreen`. Nothing new, just renamed